<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Конвертация данных операторов из CSV =&gt; MsSql, SqLite</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>550</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Настройки SqLite</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя файла</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SqliteFileName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartSqliteConverter">
     <property name="geometry">
      <rect>
       <x>479</x>
       <y>15</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Старт</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="SqlliteRowCounter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="SqliteFromServer">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>550</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Настройки Microsoft SQL</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>IP-адрес</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MsSqlServerName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пользователь</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MsSqlUserName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MsSqlUserPassword">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>25</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>База данных</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MsSqlDatabaseName">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>25</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartMsSqlConverter">
     <property name="geometry">
      <rect>
       <x>479</x>
       <y>25</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Старт</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QListView" name="DirectoryListView">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>15</y>
      <width>231</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBtnBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>550</width>
      <height>60</height>
     </rect>
    </property>
    <property name="title">
     <string>Управление</string>
    </property>
    <widget class="QPushButton" name="btnStopConverter">
     <property name="geometry">
      <rect>
       <x>479</x>
       <y>20</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Стоп</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSaveSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить настройки</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>550</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Настройки CSV</string>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя файла</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CsvFileName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Разделитель</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="CsvDelimiter">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="CsvRowCounter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="CsvAllRows">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>550</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Настройки Postgre SQL</string>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>IP-адрес</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PostgreServerName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>25</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пользователь</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PostgreUserName">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PostgreUserPassword">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>25</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>База данных</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="PostgreDatabaseName">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>25</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStartPostgreSqlConverter">
     <property name="geometry">
      <rect>
       <x>479</x>
       <y>25</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Старт</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
